.travis.yml

```diff
@@ -1,8 +1,9 @@
 sudo: required
 dist: xenial # ubuntu 16.04 distro
 script:
+  - make deps
   # runs  only when PR is merged
-  - make versionmaker
+  - author=`git log -1 --pretty=format:'%an'` && if [[ "$author"  != "Travis CI" ]]; then make versionmaker; fi
```

Comment:
:+1:

Reply:
Lets add note / comment about the need for this - preventing build for travis commits